x-pack/plugins/apm/server/routes/suggestions/get_suggestions_by_service_name.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ProcessorEvent } from '../../../common/processor_event';
+import { SERVICE_NAME } from '../../../common/elasticsearch_fieldnames';
+import { rangeQuery, termQuery } from '../../../../observability/server';
+import { Setup } from '../../lib/helpers/setup_request';
+
+export async function getSuggestionsByServiceName({
```

Comment:
Good idea changing this from a generic endpoint. Will probably make things easier.